### DISCUSSION TITLE
Keep some/none and number of threads

### DISCUSSION THREAD

**Discussion by cchaswell** [HALFpipe/HALFpipe#53] [Q&A] [answered]
Hi Lea,

I have two questions.
1. Does the "--keep none "option keep any of the files in the nipype output folder? Also does it delete any files from the derivatives folder? I end up deleting the nipype folder upon completion to save space but wasn't sure if the "--keep none" option would be ok. I use everything in the derivatives output.
2. For my cluster the pipeline defaults to using 4 threads. Am I correct in using the "--nipype-omp-nthreads 8" option to use 8 threads for the job?
Thank you!

**Comment by HippocampusGirl** [comment, marked as answer]
1. `--keep none` does not remove anything from the `derivatives` folder. So you should be good to go :-)
2. Yes, that sounds right.
